docs/azure/sdk/authentication/create-token-credentials-from-configuration.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Create token credentials from configuration
+description: This article describes how to create token credentials from configuration files
+ms.topic: how-to
+ms.custom: devx-track-dotnet, engagement-fy23
+ms.date: 1/16/2024
+---
+
+# Create Azure credential types using configuration files
+
+The `Microsoft.Extensions.Azure` library supports creating different <xref:Azure.Core.TokenCredential?displayProperty=fullName> types from values defined in _appsettings.json_ and other configuration files. This article describes the support for different `TokenCredential` types and how to configure the required values for each type.
```

Comment:
Consider linking to the NuGet package, we've had feedback that whenever docs mention a NuGet package - we should link to it.